irods/test/meta_test.py

```diff
@@ -646,6 +647,27 @@ def test_thoroughgoing_correctness_of_genquery_result_column_mappings__issue_642
         finally:
             del test_coll.metadata[meta_key]
 
+    def test_xml_mode_addresses_odd_metadata_characters__issue_582(self):
```

Comment:
What does `odd` mean for this test? Non-printable?

Reply:
Yes. I don't mind changing the name if nonprintable is better

Reply:
I see. The standard XML parser uses "odd", hence the test's title.

A comment inside the test explaining that odd can mean non-printable characters is good enough.

Reply:
We now have a comment saying that `'\x01'` is unprintable, but no reference to the word "odd". So, almost there...